### PR TITLE
Draft: 117

### PR DESCRIPTION
**Pull request by devsjc**
# Pull Request

## Description

Collect PVOutput.org worldwide data
Addresses issue #117 

Fixes #

## How Has This Been Tested?

Tested the script locally. No unit tests have been made (sadly) so as not to pollute the library codebase.

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

File: scripts/fetch_pv_timeseries.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    start_date: dt.datetime,
+    end_date: dt.datetime,
+):
+    if end_date < start_date:
+        sys.exit("End date cannot occur before start date")
+
+    # Create output directory if it doesn't already exist
+    os.makedirs(output_directory, exist_ok=True)
+
+    # Instantiate PVOutput library
```
Comment (peterdudfield) [inline scripts/fetch_pv_timeseries.py:72]: nice comments

File: scripts/fetch_pv_timeseries.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    pv_systems: pd.DataFrame = pd.read_csv(systemfile_path, index_col="system_id")
+
+    # Write output file
+    filename: str = pathlib.Path(systemfile_path).stem.replace("systems", "timeseries") + ".hdf"
+    pv.download_multiple_systems_to_disk(
+        system_ids=pv_systems.index,
+        start_date=start_date,
+        end_date=end_date,
+        output_filename=output_directory + "/" + filename,
+    )
```
Comment (peterdudfield) [inline scripts/fetch_pv_timeseries.py:87]: Is it working adding some logging? just so it says where the file is being saved too, but can alwasy add that later

**Comment by peterdudfield** [comment]
I've approved, if you want to merge, but happy to review again if you want to add mroe stuff to it

**Comment by peterdudfield** [comment]
Ive put the project as Google.org

File: scripts/fetch_pv_timeseries.py (hunk 2 above)
Comment (devsjc) [reply scripts/fetch_pv_timeseries.py:89]: Added logging in https://github.com/openclimatefix/pvoutput/pull/118/commits/24aab5b900cd05a63282388b36af83454f111848

**Pull request merged**